pwa/src/components/object_entities/dataTable.tsx

```diff
@@ -5,10 +5,10 @@ import { Link } from "gatsby";
 import FlashMessage from 'react-flash-message';
 
 export default function DataTable({ id }) {
-  const [data, setData] = React.useState(null);
+  const [data, setData] = React.useState<Array<Record<any, any>>>(null);
   const [context, setContext] = React.useState(null);
-  const [showSpinner, setShowSpinner] = React.useState(false);
-  const [alert, setAlert] = React.useState(null);
+  const [showSpinner, setShowSpinner] = React.useState<boolean>(false);
+  const [alert, setAlert] = React.useState<any>(null);
```

Comment:
Hier geldt hetzelfde, type `any` zegt eigenlijk niets. Wat we hier zouden kunnen doen is in het `Alert` component een type exporteren en deze type hier gebruiken.

Een `Alert` verwacht namelijk de volgende props (gekopieerd uit het bestand):

```
interface AlertProps {
    body: any; // deze zou hier volgens mij een () => JSX.Element moeten zijn
    alertClass: string;
    id?: string;
}
```

Dus dat is het type die we i.p.v. `any` willen meegeven.